Monika-After-Story/MonikaModDev
# Black and white striped pullover spritepack

I commissioned this outfit from mocca_bun, I've put the outfit in their name (I asked them and that's what they wanted). A black and white striped pullover outfit.

- I have already added gift reaction dlg
- There is a thumb already

## Testing:
- Make sure outfit looks good
- Make sure reaction dlg is fine
